setup.py

```diff
@@ -1,6 +1,6 @@
 import sys
 
-min_version, max_version = ((3, 6), "3.6"), ((3, 10), "3.10")
+min_version, max_version = ((3, 7), "3.7"), ((3, 10), "3.10")
```

Comment:
I don't think we need to hardcode out 3.6 (we use f strings so can't go earlier).  I'd just mention in the readme only 3.7-3.10 are officially supported